framer/Material-UI.framerfx/code/Checkbox.tsx

```diff
@@ -34,9 +36,9 @@ export function Checkbox(props) {
 
 Checkbox.defaultProps = {
   checked: false,
-  color: 'secondary',
+  color: 'secondary' as 'secondary',
```

Comment:
What does this do?

Reply:
TypeScript has a concept of "type widening" where it sometimes "widens" specific types to more general types. For example `let number = 1;` could be inferred as the literal `1` but it's very unusual that you want that here so TypeScript widens it to `number`.

It does the same thing for object properties. The type of the default props (without casting) would be `{ color: string }`. The issue here is that we only accept known literals in `color` (`"primary" | "secondary")` so we need to tell TypeScript that it should not widen the type here.

They recently added a special keyword for this `as const` but this is only available in TypeScript 3.4. Repeating the literal is basically a "poor mans `as const`" for older TypeScript versions. Since these are auto-generated it's less likely you make a mistake when refactoring e.g.

```diff
- 'secondary' as 'secondary'
+ 'primary' as 'secondary'
```
which is pretty confusing.

https://github.com/mui-org/material-ui/pull/21868/files#diff-da1f07370c6541a26f1bdc9e95e67560R144-R153 is responsible for this change.